tests/scattering1d/test_keras_scattering1d.py

```diff
@@ -16,8 +16,8 @@ def test_Scattering1D():
       buffer = io.BytesIO(f.read())
       data = np.load(buffer)
   x = data['x']
-  J = data['J']
-  Q = data['Q']
+  J = int(data['J'])
+  Q = int(data['Q'])
```

Comment:
When testing, J and Q loaded from the `test_data_1d.npz` file are `array(6, dtype=int64)` and `array(16, dtype=int64)` , respectively, not integers.

Reply:
I'm confused on why this needed to be added. Tests have been working, and it looks like they are integers according to what you state; `dtype=int64`.

Reply:
They are not intergers. `array(16, dtype=int64)` is a numpy ndarrary with one integer elment.